concerto/testing/create_carveme_mediadb_df.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd 
+def generate_mediadb(fname) : 
+    """
+    Returns: 
+        df : a dataframe of combined metabolites and
+        carbon sources, per minimal media source
+    """
+    #load carbonless minimal media file
+    dfcmin = pd.read_csv(N-less_media.csv)
```

Comment:
This function does not have the right inputs or outputs, and dfcmin should be removed.